zstacklib/zstacklib/storage/lvm/config.py

```
@@ -0,0 +1,143 @@
+
+    cmd = shell.ShellCmd(
+        f"lvmconfig --type default > {LVM_CONFIG_PATH}/lvm.conf; "
+        f"lvmconfig --type default > {LVM_CONFIG_PATH}/lvmlocal.conf"
+    )
+    cmd(is_exception=False)
+
+
+def config_lvm_by_sed(keyword: str, entry: str, files: List[str]) -> None:
+    """Configure LVM using sed replacement."""
+    if not os.path.exists(LVM_CONFIG_PATH):
+        raise Exception(f"can not find lvm config path: {LVM_CONFIG_PATH}, config lvm failed")
```

Ruff (0.15.2)

[warning] Create your own exception

(TRY002)